projects/pastanaga/src/lib/expand/expand-list.component.ts

```diff
@@ -1,11 +1,44 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, Input, HostListener } from '@angular/core';
 
 @Component({
     selector: 'pa-expand-list',
     templateUrl: 'expand-list.component.html',
     styleUrls: ['./expand-list.component.scss']
 })
 
-export class ExpandListComponent implements OnInit {
-    ngOnInit() { }
+export class ExpandListComponent {
+    @Input() large = false;
```

Comment:
Nice to have: use getter/setter to be able to have this input in the DOM without saying it's true:
`<pa-expand-list large>` instead of `<pa-expand-list [large]="true">`